test/eme.test.js

```diff
@@ -0,0 +1,157 @@
+import QUnit from 'qunit';
+
+import {standard5July2016} from '../src/eme';
+
+QUnit.module('videojs-contrib-eme eme');
+
+QUnit.test('5 July 2016 lifecycle', function(assert) {
+  assert.expect(40);
+
+  let done = assert.async();
+  let requestMediaKeySystemAccessCallback;
+  let requestMediaKeySystemAccessCallCount = 0;
+
+  navigator.requestMediaKeySystemAccess = (keySystem, options) => {
+    requestMediaKeySystemAccessCallCount++;
+    return new Promise((resolve, reject) => {
+      requestMediaKeySystemAccessCallback = resolve;
+    });
+  };
+
+  let getCertificateCallCount = 0;
+  let getCertificateCallback;
+  let getCertificate = (options, callback) => {
+    getCertificateCallCount++;
+    getCertificateCallback = callback;
+  };
+  let getLicenseCallCount = 0;
+  let getLicenseCallback;
```

Comment:
Something about these names are all blending together because they differ by only the very last few letters!

```
*CallCount
*Callback
```

Not something pressing but it adds to the cognitive overhead when reading these tests!